components/map/Map.tsx

```diff
@@ -77,29 +76,9 @@ const Map = forwardRef(
     const { markersData, zonesData, udrData, odpData } = useFilteredMapData(processedData, filters)
 
     useEffect(() => {
-      onZoneChange?.((selectedPolygon?.properties as MapUdrZone) ?? null)
+      onZoneChange?.(selectedPolygon ? selectedPolygon?.properties : null)
```

Comment:
This does not work?
```js
onZoneChange?.(selectedPolygon?.properties ?? null)
```

Reply:
it was causing problems with the type assertion but now its removed so I will change it back to `??`